tox.ini

```diff
@@ -141,4 +141,4 @@ setenv =
   PYTHONPATH = {toxinidir}{/}install
 ; Exclude src because we test the installed package, not the source tree.
 ; Without the exclude, mypy would complain about duplicate packages.
-commands = python -m mypy . --exclude '^src/'
+commands = python -m mypy . --exclude '^src/'
```

Comment:
revert whitespace change or else we may get conflicts during next copier update?